wallet/lib/pages/detailed_asset_view/owner_view_view_model.dart

```diff
@@ -202,7 +202,7 @@ class OwnerViewViewModel extends ChangeNotifier {
     );
 
     if (likesCountEither.isLeft()) {
-      LocaleKeys.something_wrong.tr().show();
+      // LocaleKeys.something_wrong.tr().show();
```

Comment:
why not remove?

Reply:
Will enable it back once the big dipper start working again.